### PR TITLE
remove some redundant scoped variables

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: src/NuGetGallery.Services/AccountManagement/DeleteAccountService.cs [hunk 1]
```diff
@@ -373,7 +373,7 @@ private async Task<DeleteAccountStatus> RunAccountDeletionTask(Func<Task> getTas
                 await _featureFlagService.RemoveUserAsync(userToBeDeleted);
                 await RemoveSupportRequests(userToBeDeleted);
 
-                using (var strategy = new SuspendDbExecutionStrategy())
+                using (new SuspendDbExecutionStrategy())
```
Comment (erdembayar) [inline src/NuGetGallery.Services/AccountManagement/DeleteAccountService.cs:376]: IMO `using (_ = new SuspendDbExecutionStrategy())` is more readable and give visibility to the the user for intention that we're discarding. Just new `using (new SuspendDbExecutionStrategy())` looks bit awkward, I don't remember seeing such code before.
Same for all others in this PR.

Comment (SimonCropp) [reply src/NuGetGallery.Services/AccountManagement/DeleteAccountService.cs:376]: but it is not discarded. it is explicitly scoped for the life of the using. having a _ discard is actively misleading the reader

Comment (erdembayar) [reply src/NuGetGallery.Services/AccountManagement/DeleteAccountService.cs:376]: In think this code itself is not designed intuitive, not sure which one is really best, probably just leave as it's better option.

Comment (SimonCropp) [reply src/NuGetGallery.Services/AccountManagement/DeleteAccountService.cs:376]: and this convention already exists in several places in the codebase

```
using (Trace.Activity(String.Format(CultureInfo.CurrentCulture, "Updating Document: {0}", updateTerm.ToString())))
```

https://github.com/NuGet/NuGetGallery/blob/dev/src/NuGetGallery/Infrastructure/Lucene/LuceneIndexingService.cs#L124

Comment (joelverhagen) [reply src/NuGetGallery.Services/AccountManagement/DeleteAccountService.cs:376]: I agree with @SimonCropp that this is an okay change. I prefer the `using (new SuspendDbExecutionStrategy())` syntax, personally. It is an aesthetic thing so it not surprising that folks will have a variety of perspectives on this. I think it's easier to just accept the PR along with the others and not deliberate very much on this. But that's my 2 cents :)

**Comment by SimonCropp** [comment]
anymore feedback on this?

File: src/NuGetGallery.Services/AccountManagement/DeleteAccountService.cs (hunk 1 above)
Comment (erdembayar) [reply src/NuGetGallery.Services/AccountManagement/DeleteAccountService.cs:376]: Yes, it's okay change but doesn't add any value to developer or user. I would refrain from this PR make code less readable.

**Review by erdembayar — APPROVED** [review]

**Pull request merged**